packages/config/src/index.ts

```diff
@@ -250,6 +250,7 @@ export default async (
     { registry: 'https://registry.npmjs.org/' },
     ...[...npmConfig.list].reverse(),
     cliOptions,
+    cliOptions['target_arch'] ? { 'target-arch': cliOptions['target_arch'] } : null,
```

Comment:
if passed `--targer_arch=x64` here, the rawConfig here will be `target_arch`, but the correct is `target-arch`